app/controllers/katello/api/v2/subscriptions_controller.rb

```diff
@@ -113,6 +113,10 @@ def upload
     api :PUT, "/organizations/:organization_id/subscriptions/refresh_manifest", N_("Refresh previously imported manifest for Red Hat provider")
     param :organization_id, :number, :desc => N_("Organization id"), :required => true
     def refresh_manifest
+      if @organization.manifest_expired?
+        fail HttpErrors::BadRequest, _("This Organization's subscription manifest has expired. A new one should be imported.")
```

Comment:
Not too sure about the wording. Any better ideas?

Reply:
I think the first sentence is okay, but what do you think about "Please import a new manifest." for the second?